hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -109,10 +109,8 @@ public HddsDispatcher(ConfigurationSource config, ContainerSet contSet,
     this.containerCloseThreshold = conf.getFloat(
         HddsConfigKeys.HDDS_CONTAINER_CLOSE_THRESHOLD,
         HddsConfigKeys.HDDS_CONTAINER_CLOSE_THRESHOLD_DEFAULT);
-    this.isBlockTokenEnabled = conf.getBoolean(
-        HddsConfigKeys.HDDS_BLOCK_TOKEN_ENABLED,
-        HddsConfigKeys.HDDS_BLOCK_TOKEN_ENABLED_DEFAULT);
-    this.tokenVerifier = tokenVerifier;
+    this.tokenVerifier = tokenVerifier != null ? tokenVerifier
```

Comment:
NIT: can we move new NoopTokenVerifier() to the caller in OzoneContainer?

Reply:
I think checking for `null` here is more defensive.  Also, `HddsDispatcher` is created by several tests, I wanted to avoid changing those.

Reply:
I found that too from later code changes in tests. Good to be defensive here.